modules/xmpp/JingleSessionPC.js

```diff
@@ -1855,6 +1855,9 @@ export default class JingleSessionPC extends JingleSession {
                 if (track) {
                     logger.debug(`Existing SSRC ${s.ssrc}: new owner ${s.owner}. name=${s.source}`);
 
+                    // Update the SSRC owner.
+                    this._signalingLayer.setSSRCOwner(s.ssrc, s.owner);
```

Comment:
Can we rename the method? I think this one stopped to be a getter a long time ago :)

Another option would be to somehow split it in 2 functions - 1 getter and 1 that actually modifies the state.

Reply:
I meant the method where this line is executed - `getNewSources`

Reply:
Not very relevant to the PR but here you add more set operations...